### PR TITLE
Implementation of Bead sort in ruby

### LINKED ISSUE #1751: Implementation of bead sort in c++ and ruby
I want to implement the bead sort in c++  and ruby.
And also modify the documentation.There is also no documentation about bead sort.
There is no implementation of them.

### PR DESCRIPTION
**Pull request by sparkingdark**
Fixes #1751 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Implementation Bead sort in ruby
- 

#### Languages Used:
- Ruby
- 

#### Files Added:
- Bead_Sort.rb 
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by darkdebo** [comment]
Please check it @jainaman224

**Comment by darkdebo** [comment]
@shlok007  i use rufo format the code.Please check it out.

**Comment by shlok007** [comment]
Looks good.

**Comment by darkdebo** [comment]
thanks....

**Comment by darkdebo** [comment]
please review it @shlok007

**Comment by darkdebo** [comment]
Can anyone review it....  @jainaman224 @shlok007

File: Bead_Sort/Bead_Sort.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Bead Sort is a Natural Sorting Algorithm Also known as Gravity sort,
+# this algorithm was inspired from natural phenomenons and was designed keeping in mind objects(or beads) falling under the influence of gravity.
```
Comment (somya-kapoor) [inline Bead_Sort/Bead_Sort.rb:2]: Please break the sentence into two-line.

**Comment by somya-kapoor** [comment]
Are you working on it?

**Comment by sparkingdark** [comment]
Yes due to some network issues I can't focused on working but I do the updated as soon as possible.

**Pull request merged**